tests/API0/testclassselector.py

```diff
@@ -47,7 +47,7 @@ def test_multiple_class_instance_constructor2(self):
         self.assertEqual(p.g, 'A')
 
     def test_multiple_class_instance_error(self):
-        exception = "ClassSelector parameter 'g' value must be an instance of \(int, str\), not 3.0."
+        exception = r"ClassSelector parameter 'g' value must be an instance of \(int, str\), not 3.0."
```

Comment:
The backslashes are required since the string is handled as a regex by 'assertRaisesRegexp', with those the tests would fail.

Reply:
As for the set notation instead of parentheses, that sounds like a feature request ;) I'm fine with the current state, see e.g. `isinstance(x, (int, str))`.

Reply:
Ok, makes sense.